static-catalog/examples/CBERS/CBERS_4_MUX_20170528_090_084_L2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "CBERS_4_MUX_20170528_090_084_L2", 
+  "type": "Feature", 
+  "bbox": [
+    13.700498, 
+    23.465111, 
+    14.98818, 
+    24.812825
+  ], 
+  "geometry": {
+    "type": "MultiPolygon", 
+    "coordinates": [
+      [
+        [
+          [
+            13.891487, 
+            23.463987
+          ], 
+          [
+            13.723733, 
+            24.567134
+          ], 
+          [
+            14.786396, 
+            24.811268
+          ], 
+          [
+            14.95489, 
+            23.703021
+          ], 
+          [
+            13.891487, 
+            23.463987
+          ]
+        ]
+      ]
+    ]
+  }, 
+  "properties": {
+    "datetime": "2017-05-28T09:01:17Z", 
+    "provider": "INPE", 
+    "license": "CC-BY-SA-3.0"
```

Comment:
Would be great to add some additional fields here from the metadata. The recommendation is to prefix them, see https://github.com/radiantearth/stac-spec/blob/master/static-catalog/static-recommendations.md#prefixes

@matthewhanson is working on the 'EO profile', which should give some common fields / definitions for things like sun angle, view angle, etc. But in the meantime you could just use a provider prefix, like inpe:satellite_name, inpe:orientatationAngle, etc.

I'd say add the fields that could make sense for users to search. The rest can live in extended metadata.